starknet_py/tests/e2e/docs/code_examples/test_websocket_client.py

```diff
@@ -210,39 +206,54 @@ def handler(transaction_status_notification: TransactionStatusNotification):
     assert unsubscribe_result is True
 
 
-@pytest.mark.skip(
-    reason="TODO(cptartur): Unskip when adding changes for websockets in RPC 0.9.0 "
-)
 @pytest.mark.asyncio
-async def test_subscribe_pending_transactions(
+async def test_subscribe_new_transaction_receipts(
     websocket_client: WebsocketClient,
     deployed_balance_contract,
     argent_account_v040: BaseAccount,
 ):
     account = argent_account_v040
-    pending_transactions: List[Union[int, Transaction]] = []
-
-    # docs-start: subscribe_pending_transactions
-    from starknet_py.net.websockets.models import PendingTransactionsNotification
-
-    # Create a handler function that will be called when a new pending transaction is emitted
-    def handler(pending_transaction_notification: PendingTransactionsNotification):
-        # Perform the necessary actions with the new pending transaction...
-
-        # docs-end: subscribe_pending_transactions
-        nonlocal pending_transactions
-        pending_transactions.append(pending_transaction_notification.result)
-
-    # docs-start: subscribe_pending_transactions
-    # Subscribe to pending transactions notifications
-    subscription_id = await websocket_client.subscribe_pending_transactions(
-        handler=handler,
+    transaction_receipts: List[TransactionReceipt] = []
+    transactions = []
+
+    # docs-start: subscribe_new_transaction_receipts
+    from starknet_py.net.websockets.models import NewTransactionReceiptsNotification
+
+    # Create a handler function that will be called when a new transaction is emitted
+    def handler_a(
+        new_transaction_receipts_notification: NewTransactionReceiptsNotification,
+    ):
+        # Perform the necessary actions with the new transaction receipts...
+
+        # docs-end: subscribe_new_transaction_receipts
+        nonlocal transaction_receipts
+        transaction_receipts.append(new_transaction_receipts_notification.result)
+
+    def handler_b(
+        new_transaction_notification: NewTransactionNotification,
+    ):
+        # Perform the necessary actions with the new transaction receipts...
+
+        # docs-end: subscribe_new_transaction_receipts
+        nonlocal transactions
```

Comment:
Originally I added a second handler for debugging. If you think it's beneficial to have more elaborate example, let's keep it but it's not critical for me.

Reply:
Removed